_posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "Partner Highlight: Oracle Cloud Infrastructure Search Service with OpenSearch is GA!"
+authors:
+  - andhopp
+  - vachshah
+  - dtaivpp
+date: 2022-05-10
+categories:
+  - partners
+
+excerpt: "Oracle has released it's Cloud Infrastructure Search Service!"
```

Comment:
Let's add "...with OpenSearch" in the excerpt.

Reply:
Added